api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java

```diff
@@ -192,7 +191,7 @@ private void insertPatientStubIfNeeded(Patient patient) {
 		criteriaExactMatch = new PatientSearchCriteria(sessionFactory, criteriaExactMatch).prepareCriteria(query, true,
 		    true, includeVoided);
 		
-		Set<Patient> patients = new LinkedHashSet<Patient>();
+		Set<Patient> patients = new LinkedHashSet<>();
```

Comment:
@Nishen-Peiris oh i see! :)
The ticket under which you are creating these pull requests does not seem related to the changes you are making. Do you mind creating a new ticket? :)